tensorflow_similarity/architectures/resnet18.py

```diff
@@ -175,7 +175,7 @@ def block0(
         kernel_initializer=tf.keras.initializers.LecunUniform(),
         name=name + "_2_conv",
     )(x)
-    x = tf.keras.layers.BatchNormalization(
+    x = tf.keras.layers.experimental.SyncBatchNormalization(
```

Comment:
Since the other two build functions now return models, should we also have this return a model as well? It would be best if these were all consistent across the different architectures.